Assets/MixedRealityToolkit.Tests/PlayModeTests/SolverTests.cs

```diff
@@ -15,14 +15,16 @@
 using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
 using NUnit.Framework;
 using System.Collections;
+using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.TestTools;
 
 namespace Microsoft.MixedReality.Toolkit.Tests
 {
-    public class SolverTests : BasePlayModeTests
+    public class SolverTests
```

Comment:
instead of breaking inheritance, can you just add virtual to the BasePlayModeTests function and then override here?